tmail-backend/blob/blobid-list/src/main/java/com/linagora/tmail/blob/blobid_list/cassandra/CassandraSingleSaveBlobStoreDAO.java

```diff
@@ -0,0 +1,37 @@
+package com.linagora.tmail.blob.blobid_list.cassandra;
+
+import com.linagora.tmail.blob.blobid_list.SingleSaveBlobStoreDAO;
+import org.apache.james.backends.cassandra.init.configuration.CassandraConfiguration;
+import org.apache.james.blob.api.BlobId;
+import org.apache.james.blob.api.BucketName;
+import org.apache.james.blob.cassandra.CassandraBlobStoreDAO;
+import org.apache.james.blob.cassandra.CassandraBucketDAO;
+import org.apache.james.blob.cassandra.CassandraDefaultBucketDAO;
+import reactor.core.publisher.Mono;
+
+import javax.inject.Inject;
+
+public class CassandraSingleSaveBlobStoreDAO extends CassandraBlobStoreDAO {
+
+    private final SingleSaveBlobStoreDAO singleSaveBlobStoreDAO;
```

Comment:
I have circle dependency here. It makes me feel vicious.
CassandraSingleSaveBlobStoreDAO -> SingleSaveBlobStoreDAO
SingleSaveBlobStoreDAO -> BlobStoreDAO
CassandraBlobStoreDAO extend BlobStoreDAO
CassandraSingleSaveBlobStoreDAO extend CassandraBlobStoreDAO

Reply:
I missed here
I need only `S3SingleSaveBlobStoreDAO`